algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -463,7 +464,7 @@ case class SparseHLL(override val bits: Int, maxRhow: Map[Int, Max[Byte]]) exten
         val newRhoW = reducedMaxRhoW.getOrElse(newJ, 0: Byte)
         reducedMaxRhoW += (newJ -> (newRhoW.max(modifiedRhoW)))
     }
-    SparseHLL(reducedBits, reducedMaxRhoW.toMap.mapValues(Max(_)))
+    SparseHLL(reducedBits, reducedMaxRhoW.toMap.view.mapValues(Max(_)).toMap)
```

Comment:
can we do `reducedMaxRhoW.iterator.map { case (k, v) => (k, Max(v)) }.toMap` here? looks like we have two conversions to map here.

Reply:
ahh I missed this one 🤦‍♂